boutiques_descriptors/deepprep_24_1_2.json

```diff
@@ -2,9 +2,9 @@
     "name": "DeepPrep",
     "tool-version": "24.1.2",
     "schema-version": "0.5",
-    "author": "personalized Brain Functional Seuctors (pBFS) Lab",
+    "author": "personalized Brain Functional Sectors (pBFS) Lab",
     "description": "An accelerated, scalable, and robust pipeline for neuroimaging preprocessing empowered by deep learning",
-    "descriptor-url": "https://deepprep.readthedocs.io/en/latest/",
+    "descriptor-url": "https://deepprep.readthedocs.io/en/24.1.2/",
```

Comment:
BTW 
I think it is wrong url, descriptor-url is the url of the boutiques descriptor not tool